Fix realpath and tac missing on OSX

davidxbors opened the pull request:
As @Sergiu121 pointed out in issue #1 `tac` is named `gtac` on macOS systems when installed with the `coreutils` package using MacPorts. The same problem exists with `realpath`. A simple solution is to install the `coreutils` package using `brew` instead.

davidxbors committed:
Fix realpath and tac missing on OSX

As @Sergiu121 pointed out in issue #1 `tac` is named `gtac`
on macOS systems when installed with the `coreutils` package
using `MacPorts`. The same problem exists with `realpath`.
A simple solution is to install the coreutils package using
`brew` instead.

razvand commented on README.md:
````diff
@@ -46,18 +46,19 @@ sudo apt-get install coreutils grep sed tar wget unzip build-essential
 ```
 
-or the following commands on macOS using [MacPorts](https://www.macports.org/):
+or the following commands on macOS using [MacPorts](https://www.macports.org/) and [Homebrew](https://brew.sh/):
````
Isn't there a `coreutils` package as part of MacPorts?

davidxbors replied:
I provided a description to the commit that answers your question.

razvand reviewed (changes requested):
Please update the commit message. The commit line looks good. But you need a more detailed description about what the commit does. Mention the requirement for the `coreutils` package to include the `realpath` and the `tac` executable files.

davidxbors committed:
Make brew the default package manager

Make brew the default package manager in the README install instructions
for Darwin systems.

razvand reviewed (approved):
All good. Thanks, @davidxbors
